### PR TITLE
core(is-on-https): add Mixed Content verbiage to description

### PR DESCRIPTION
**Pull request by Beytoven**
Adding a line about Mixed Content to signal that it's covered by this audit.

### CODE REVIEW

File: lighthouse-core/audits/is-on-https.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ const UIStrings = {
   failureTitle: 'Does not use HTTPS',
   /** Description of a Lighthouse audit that tells the user *why* HTTPS use is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'All sites should be protected with HTTPS, even ones that don\'t handle ' +
-      'sensitive data. HTTPS prevents intruders from tampering with or passively listening ' +
+      'sensitive data. Sites should avoid having Mixed Content, where some resources are ' +
+      'loaded over HTTP despite the initial request being over HTTPS. HTTPS prevents ' +
+      'intruders from tampering with or passively listening ' +
```
Comment (patrickhulce) [inline lighthouse-core/audits/is-on-https.js:22]:
```suggestion
      'sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content), ' +
      'where some resources are loaded over HTTP despite the initial request being served over HTTPS. ' +
      'HTTPS prevents intruders from tampering with or passively listening ' +
```

WDYT? might need to make some line length adjustments its hard to tell in GH

if you're not a fan of the wording changes, the "being *served* over" and the link would still be great :)

File: lighthouse-core/audits/is-on-https.js [hunk 2]
```diff
@@ -17,7 +17,9 @@ const UIStrings = {
   failureTitle: 'Does not use HTTPS',
   /** Description of a Lighthouse audit that tells the user *why* HTTPS use is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```
Comment (patrickhulce) [inline lighthouse-core/audits/is-on-https.js:18]:
```suggestion
  /** Description of a Lighthouse audit that tells the user *why* HTTPS use *for all resources* is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```

just some extra emphasis for the translators to keep the mixed content bit important perhaps?

File: lighthouse-core/audits/is-on-https.js (hunk 1 above)
Comment (Beytoven) [reply lighthouse-core/audits/is-on-https.js:22]: I like it, especially the link.

**Comment by patrickhulce** [comment]
I think it needs a sample json update, but looks good :)

**Pull request merged**